indoorplants/validation/curves.py

```diff
@@ -14,6 +14,9 @@
 def validation_curve(X, y, score, model_type, param_name,
                      param_range, other_params={},
                      semilog=False, figsize=(11, 8),
+                     matplotlib_kwargs_xlabels={},
```

Comment:
@xcao38 this looks great in theory, but there's a danger in setting _mutable_ default values - see [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). Better is to do one of the following:

- set default to `None` and then update to `{}` if `None`
- lump them all into `**kwargs`, which will auto-create an empty dict if nothing is passed

the latter of these options is obviously nice, but it's kind of hard when, as is the case here, we have different functions / objects we're trying to allow passing varying amounts of arguments too!

maybe it'd be worth thinking about exactly where we _ really need_ to expose `**kwargs` ? to me, the core plot seems important, for `rot` and `alpha`, but maybe we can let the others go? I may be wrong, though - what do you think?